getting-started/dcos/linkerd-package-options.json

```diff
@@ -0,0 +1,13 @@
+{
+  "linkerd": {
+    "admin-port": 9990,
+    "config-filename": "linkerd.yaml",
+    "config-uri": "https://raw.githubusercontent.com/BuoyantIO/linkerd-examples/master/getting-started/dcos",
+    "cpus": 0.25,
+    "instances": 1,
+    "mem": 256.0,
+    "name": "linkerd-example",
+    "routing-port": 4140,
+    "resource-role": "slave_public"
```

Comment:
i'm realizing the linkerd universe package is still on 0.6.0, so i'll have to update anyway. let me add `resource-role` and make this demo a real thing.